fgpyo/sequence.py

```diff
@@ -88,9 +99,55 @@ def longest_hp_length(bases: str) -> int:
     return max_hp
 
 
-def gc_content(bases: str) -> float:
-    """Calculates the fraction of G and C bases in a sequence."""
-    if len(bases) == 0:
+def longest_dinucleotide_run_length(bases: str) -> int:
```

Comment:
I think this function (and longest multi-nuc run) need to be clear on how they handle case, and ideally should be case-insensitive.  E.g. if I pulled sequence from a FASTA file and it's `ACGACCATATatatatatatGC` I would want it to count all of `ATATatatatatat`.